### PR TITLE
Print the malioc_diff.py command to update for new shaders

### PR DESCRIPTION
**Pull request by zanderso**

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM!

File: impeller/tools/malioc_diff.py [hunk 1]
```diff
@@ -285,14 +285,20 @@ def main(argv):
...
-          .format(filename)
-      )
+      new_shader = True
+      print('Shader {} is new.'.format(filename))
       changed = True
 
+  if new_shader:
+    print(
+        'There are new shaders. The golden file must be updated with:\n\n'
+        '$ ./flutter/impeller/tools/malioc_diff.py --before {} --after {} --update'
```
Comment (gaaclarke) [inline impeller/tools/malioc_diff.py:298]: I believe there is a precondition that `android_debug_unopt` is built with the `--malioc-path` path for this script to work which is worth mentioning.

Comment (gaaclarke) [reply impeller/tools/malioc_diff.py:298]: Or maybe it's better to put that into the script itself.

Comment (zanderso) [reply impeller/tools/malioc_diff.py:298]: Good call. Updated.

**Review by gaaclarke — APPROVED** [review]

**Pull request merged**